Scripts/DowloadPortalItemData.swift

```diff
@@ -245,31 +246,46 @@ var downloadedItems = previousDownloadedItems
 let dispatchGroup = DispatchGroup()
 
 portalItems.forEach { portalItem in
```

Comment:
[`forEach`](https://forums.swift.org/t/pitch-mutating-foreach/65950/10) can be replaced in the follow up PR.